transactional_licence_form/forms.py

```diff
@@ -133,51 +134,51 @@ class ProjectPurposeForm(FCLForm):
 
 class PublicStatementForm(FCLForm):
 
-    title = "Public Statement"
+    title = "Step 5 - Public Statement"
 
     def layout(self):
         return Layout(Field.textarea("public_statement", max_words=150))
 
     public_statement = fields.FCLCharField(
-        label="14. Please provide a public statement",
-        help_text="Please aim for no more than around 150 words.",
-        widget=forms.Textarea(attrs={"data-maxwords": 150}),
+        label="15. Please provide a public statement",
+        help_text="Please aim for 150 words",
+        widget=forms.Textarea(attrs={"maxlength": 1500}),
```

Comment:
If you aimed for 150, got to 200, you're probably ok with six-letter words on average...

Reply:
This is to match the configuration on the receiving end of the email.